### ISSUE TITLE
ClearableValue is not working for me.

### ISSUE THREAD

**Issue by chobo2** [JedWatson/react-select#1507]
Hi

I am having problems with the little x in the select bar. I tried to leave it and I tried to disable it and both don't work.

When I hit the x it just errors out saying "null value"

```
      var formatTypeOptions = [
            { value: 'CSV', label: 'CSV', clearableValue: false },
            { value: 'XML', label: 'Excel', clearableValue: false }
        ];

    <Select id="formats" placeholder="choose"
                            name="formats"
                            value={this.state.format}
                            options={formatTypeOptions}
                            onChange={(event) => this.formatChange(event)}
                            />

   formatChange(event) {
        this.setState({ hide: false, format: event.value, hasErrors: false });
    }
```

First I set it clearableValue to false so I would think it would not show up but it does, when I click on it and fails.

**Comment by VanCoding** [comment]
I think my pull request #1508 might fix a part of this.

**Comment by agirton** [comment]
@chobo2 I don't see a `multi` prop on your `Select` component. This is needed for clearableValue to work: https://github.com/JedWatson/react-select#multiselect-options

**Comment by JedWatson** [comment]
Should be fixed by #1508, please reopen if you can still replicate this after that fix.

Thanks!
